### ISSUE TITLE
Nav to marked terminal error

### ISSUE THREAD

**Issue by peleusj** [ThePrimeagen/harpoon#52]
Have an issue when navigate.
When I first add 2 marks, one file and one terminal, navigation works just fine.
But when I close nvim, and then reopen it, there is an issue. 
I can still navigate to the file normally, but I can only navigate to marked terminal for one time.
When I try to naviage from file to marked terminal again, it has an error.
<img width="892" alt="Screen Shot 2021-04-01 at 1 21 54 PM" src="https://user-images.githubusercontent.com/17824110/113247392-8dde6980-92ed-11eb-98ce-be5c6651e9fb.png">
<img width="1261" alt="Screen Shot 2021-04-01 at 1 23 57 PM" src="https://user-images.githubusercontent.com/17824110/113247403-933bb400-92ed-11eb-9612-f1c396ca449a.png">

**Comment by asbjornhaland** [comment]
I don't think we have though about using marks for terminals. Have you seen the `term` part of this plugin?

**Comment by ThePrimeagen** [comment]
Interesting 

I can solve this, but there is a terminal specific commands to!

Let me try this shortly to fix it
